internal/metrics/metrics.go

```diff
@@ -248,6 +250,28 @@ func (t *TSDB) MaxTime() time.Time {
 	return fromDbTime(t.db.Head().MaxTime())
 }
 
+// HeadMinTime
+func (t *TSDB) HeadMinTime() time.Time {
+	return fromDbTime(t.db.Head().MinTime())
+}
+
+// BlockMinMax
+func (t *TSDB) BlocksMinMax() []time.Time {
```

Comment:
This function is 0 tested, I think that should be tested in `internal/metrics_test.go`

Reply:
it is a very small and simple one. It's not worth the effort. Actually, there are others such as AddVector that were not tested. What test cases would u suggest?